### ISSUE TITLE
Date Picker does not open anymore

### ISSUE THREAD

**Issue by mxmtsk** [xgfe/react-native-datepicker#135]
The datepicker has worked fine all the time but suddenly stopped working. I don't know what I have changed, if I click in the date field, `onOpenModal` does get triggered but no datepicker is shown. no error is thrown either. How to debug now?

```
      <View style={{ paddingBottom: 25 }}>
        {locals.label ? <Text style={FormStyles.controlLabel.normal}>{locals.label}</Text> : null}
        <DatePicker
          style={{ flex: 1, minWidth: width - 30 }}
          date={locals.value}
          mode="datetime"
          placeholder="select date"
          format="YYYY-MM-DD HH:mm"
          minDate={moment().format('YYYY-MM-DD HH:mm')}
          confirmBtnText="Confirm"
          onOpenModal={() => { console.log('opened'); }}
          cancelBtnText="Cancel"
          customStyles={{
            dateIcon: {
              position: 'absolute',
              left: 0,
              top: 4,
              marginLeft: 0,
            },
            dateInput: {
              marginLeft: 36,
              flex: 1,
              borderRadius: 3,
            },
            dateText: {
              color: '#fff',
            },
            placeholderText: {
              color: '#fff',
            },
          }}
          onDateChange={(date) => {
            console.log(date);
            // this.props.onChangeDate(date);
          }}
        />
      </View>
```

**Comment by rbaron** [comment]
I struggled a while with this too. In my case (react-native@0.42), the problem was related to this issue: https://github.com/facebook/react-native/issues/12515.

What worked for me: disabling remote debug. It seems to be resolved in in 0.42.2, so upgrading might help as well.

**Comment by mxmtsk** [comment]
@rbaron You are right, it works when I disable the remote debugger. Thank you SO much. This caused a big headache!
